machine_learning/classifier.py

```diff
@@ -0,0 +1,127 @@
+import sys
+import pandas as pd
+import numpy as np
+from sklearn.metrics import precision_recall_fscore_support
+from sklearn.metrics import accuracy_score
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.datasets import make_classification
+
+from gensim.models.keyedvectors import KeyedVectors
+model = KeyedVectors.load_word2vec_format("glove_model2.txt", binary=False, limit=50000)
+
+
+def feature_extractor(orig_object, data_object):
+    """
+    This function is used to extract features in the data set and return a list of all the features computed.
+    :param orig_object:
+    :param data_object:
+    :return feat(list):
+    """
+    feat = []
+
+    plant_similarity_score = model.similarity(str(orig_object[1]),str(data_object[1]))
+    feat.append(plant_similarity_score)
+
+    storage_similarity_score = model.similarity(str(orig_object[2]),str(data_object[2]))
+    feat.append(plant_similarity_score)
+
+    material_group_score = int(str(orig_object[3]))- int(str(data_object[3]))
+    score = 0
```

Comment:
Is assigning `score` a value here necessary?

Reply:
you need to initialise score to some value atleast